docs/guides/Screen-Nav-Options.md

```diff
@@ -96,7 +96,23 @@ The 2nd argument passed to the function are the default values for the `header`
 
 ## Tab Navigation Options
 
-Coming Soon
+```js
+class TabScreen extends React.Component {
+  static navigationOptions = {
+    label: 'Tab Label',
```

Comment:
I'm not 100% sure, but don't these properties (`lable`, `icon`, `visible`) have to be nested into another property `tabBar`?